network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/quote_block.html

```diff
@@ -1,15 +1,16 @@
+{% extends "./base_streamfield_block.html" %}
 {% load wagtailcore_tags %}
 
-<div class="row my-default">
-  <div class="col">
-    {% for quote in value.quotes %}
-    {% comment %} Next task for this component is to handle multiple quotes. For now, let's use just the first {% endcomment %}
-    {% if forloop.counter is 1 %}
-    <div class="feature-quote">
-      <h3>{{quote.quote}}</h3>
-      <p class="h6-heading">{{quote.attribution}}</p>
-    </div>
-    {% endif %}
-    {% endfor %}
+{% block block_content  %}
+  <div class="my-2">
+      {% for quote in value.quotes %}
+      {% comment %} Next task for this component is to handle multiple quotes. For now, let's use just the first {% endcomment %}
+      {% if forloop.counter is 1 %}
+      <div class="feature-quote">
+        <h3>{{quote.quote}}</h3>
+        <p class="h6-heading">{{quote.attribution}}</p>
+      </div>
+      {% endif %}
+      {% endfor %}
```

Comment:
small nit: let's un-indent that again, we're using 2-space indentation everywhere =)